js/src/Ice/Connection.d.ts

```diff
@@ -222,7 +222,7 @@ declare module "ice" {
             /**
              * The headers from the HTTP upgrade request.
              */
-            headers: HeaderDict;
+            get headers(): HeaderDict;
```

Comment:
The HeaderDictHelper sounds like a left over from when this code was generated from local Slice.